README.md

```
@@ -0,0 +1,7 @@
+# VPSM
+A worse version of (HCloud)[https://hetzner.cloud] for now.
```

markdownlint-cli2 (0.20.0)

[warning] Reversed link syntax
(HCloud)